zio-http-gen/src/test/scala/zio/http/gen/scala/CodeGenSpec.scala

```diff
@@ -975,5 +976,67 @@ object CodeGenSpec extends ZIOSpecDefault {
           }
         }
       } @@ TestAspect.exceptScala3,
+      test("Generate all responses") {
+        val oapi =
+          OpenAPI(
+            openapi = "3.0.0",
+            info = OpenAPI.Info(
+              title = "XXX",
+              description = None,
+              termsOfService = None,
+              contact = None,
+              license = None,
+              version = "1.0.0",
+            ),
+            paths = ListMap(
+              OpenAPI.Path(name = "/api/a/b") -> OpenAPI.PathItem(
```

Comment:
Path has no public constructor but only a smart constructor. Calling .get should be okay here

Reply:
Oh that compiles on `Scala213`, I guess you're not using `-Xsource:3`. I took a slightly different approach to fixing it.